### DISCUSSION TITLE
Question about L2 Cache and RAM timings

### DISCUSSION THREAD

**Discussion by PlayNeth** [86Box/86Box#5085] [Q&A]
Hello, I'd like to know if 86Box can actually emulate those two, as L2 cache doesn't seem to do anything on pretty much any of the 486 motherboards I tried, same goes for RAM timings, they don't do anything. I noticed that my benchmarks were oddly slow for the setup picked compared to ones I've seen in YT videos.

**Comment by jriwanek** [comment]
Cache implementation would actually slow things down.
